data-prepper-performance-test/src/gatling/java/org/opensearch/dataprepper/test/performance/tools/Chain.java

```diff
@@ -0,0 +1,21 @@
+package org.opensearch.dataprepper.test.performance.tools;
```

Comment:
This class, and I believe all the tools classes are missing the copyright header.